modules/tests/shared/src/test/scala/gsp/math/AngleSpec.scala

```diff
@@ -135,4 +134,29 @@ final class AngleSpec extends CatsSuite {
     }
   }
 
+  test("Difference with itself is 0") {
+    forAll { (a: Angle) =>
+      a.difference(a) shouldEqual Angle.Angle0
+    }
+  }
+
+  test("Difference with reverse is 180") {
+    forAll { (a: Angle) =>
+      a.difference(a + Angle.Angle180) shouldEqual Angle.Angle180
+    }
+  }
+
+  test("Difference is in the range [0 .. π)") {
+    implicit val order: Order[Angle] = Angle.AngleOrder
+    forAll { (a: Angle, b: Angle) =>
+      (a.difference(b) >= Angle.Angle0 && a.difference(b) < Angle.Angle180) shouldBe true
+    }
+  }
+
+  test("Difference is commutative") {
+    forAll { (a: Angle, b: Angle) =>
+      a.difference(b) shouldEqual a.difference(b)
```

Comment:
One of these should be `b.difference(a)` ;-)

Reply:
Good catch